tests/Makefile.am

```diff
@@ -50,8 +54,12 @@ testlayers_SOURCES = \
     ./common/layerrenderer.cpp \
     ./common/gllayerrenderer.cpp \
     ./common/glcubelayerrenderer.cpp \
-    ./common/videolayerrenderer.cpp \
-    ./common/imagelayerrenderer.cpp \
     ./common/esTransform.cpp \
     ./common/layerfromjson.cpp \
     ./apps/jsonlayerstest.cpp
+
+if !ENABLE_GBM
```

Comment:
That would result in making changes in VideoLayer and texture layer to avoid compilation error and I would have to do more #ifdef. So think its better to enable on what is working with GBM and minimise changes else where